.github/workflows/build.yml

```diff
@@ -49,7 +49,7 @@ jobs:
     - uses: codecov/codecov-action@v3
       name: Upload coverage to Codecov
       with:
-        files: ./src/Polly.Specs/coverage-reports/Cobertura.xml,./src/Polly.Core.Tests/coverage-reports/Cobertura.xml
+        files: ./artifacts/coverage-reports/Polly.Core.Tests/Cobertura.xml,./artifacts/coverage-reports/Polly.Specs/Cobertura.xml
```

Comment:
We will have more projects later, is it possible to have a mask here?

Reply:
I did try a directory at first https://github.com/App-vNext/Polly/pull/1076/commits/83c9f1df1f7403597ffdafca54783c9f32328e7b, but it [didn't seem to work](https://github.com/App-vNext/Polly/actions/runs/4488732817/jobs/7893695475#step:5:29).

I'll try using a wildcard and see if that works.

Reply:
Looks like that [didn't work](https://github.com/App-vNext/Polly/actions/runs/4490361008/jobs/7897328721?pr=1076#step:5:27) 😞

I'll revert to the previous change and we can look to improve it in the future.